### DISCUSSION TITLE
Running SSR application on domain that connected to Cloudflare

### DISCUSSION THREAD

**Discussion by barikhan1986** [nuxt/nuxt#9757] [Questions (Nuxt 2)]
Hello,

I'm trying to serve SSR application on domain that connected to Cloudflare, i did the following:

in nuxt.config.js

  server: {
    port: 49000,
    host: 'mydomain.com', 
  },  

when I try to run npm run start i got "listen EADDRNOTAVAIL: address not available 104.21.20.30:49000", it try to serve application on cloudflare IP, 

when i put Host:'0.0.0.0' application is serve on server IP not domain.

how i can fix this issue?

**Comment by azataiot** [comment]
1. serve the application on 0.0.0.0:49000
2. use Nginx for proxy pass.
